kds/kds.go

```diff
@@ -114,17 +127,23 @@ var (
 	}
 )
 
-// TCBVersion is a 64-bit bitfield of different security patch levels of AMD firmware and microcode.
-type TCBVersion uint64
+// TCBVersion represents the SEV-SNP TCB_VERSION
+type TCBVersion struct {
+	// Version identifies the version of this structure as defined in V[C,L]EK x509 certificate extensions
+	Version uint8
```

Comment:
Where the version is purely for internal parsing and handling, does this need to be public?

Reply:
Fair enough. There are definitely ways of obfuscating this, but it is probably outside of the scope of this PR.